client/src/cli.rs

```diff
@@ -1181,4 +1181,22 @@ pub enum DeviceSettings {
         #[arg(value_parser, action = ArgAction::Set)]
         enabled: bool,
     },
+
+    /// Configure the tap-tempo window (ms) for effect preset tapping
+    TapTempoWindow {
```

Comment:
There's an obvious question here, is this needed? The maximum possible value for the echo tempo is 300ms, so it makes sense to assume that if they're tapping slower than 300ms then we should just ignore or reset the counter.